cache/cache.go

```diff
@@ -133,8 +135,26 @@ func CopyOut(config config.Config, manifestFile string)(error){
             return err
         }
     }
+    for _, symLink := range manifestdata.SymLink{
+        os.Symlink(symLink[1], filepath.Join(config.BaseDir, symLink[0]))
+    }
+    // print out log file
+    if manifestdata.LogFile != ""{
+        logFile := filepath.Join(dirOfCachedOutputFiles, manifestdata.LogFile)
+        if utils.Exists(logFile){
+            file, err := os.Open(logFile)
+            if err != nil{
+                fmt.Printf("Failed to open %v\n", logFile)
+            }else{
+                scanner := bufio.NewScanner(file)
+                scanner.Split(bufio.ScanLines)
+                for scanner.Scan() {
+                    fmt.Println(scanner.Text())
+                }
+            }
+        }
```

Comment:
Do we have to read and print line by line. something like this https://github.com/tokuhirom/golang-samples/blob/master/samples/coreutils/cat.go ?

Reply:
took care of it in the new push